vars.yml

```diff
@@ -38,7 +111,7 @@ ibmcloud_image_ids:
     "rhel_8.4": "r042-06ad9b4d-02a7-44bb-989e-18f1882646ac"
 
 ibmcloud_vsi_profile: 'bx2-4x16'
-ibmcloud_vsi_count: 2
+ibmcloud_vsi_count: "{{ tower_control_nodes|int + tower_hybrid_nodes|int |int}}"

```

Comment:
we shouldn't do this. Please note this repo is also used by other users who spin up vm's on ibm cloud without tower being installed. For them tower_control_nodes and tower_hybrid_nodes will not have the relevance.

Reply:
ahh ok